vms/vm-under-test/scripts/customize-vm

```diff
@@ -90,21 +90,24 @@ cat <<'EOF' > "$checkup_boot_script_full_path"
 # Copyright 2024 Red Hat, Inc.
 #
 
-set -e
+set -x
 
 checkup_boot_ready_marker_full_path=$1
+checkup_tuned_adm_set_marker_full_path="/var/realtime-checkup-tuned-adm-set-marker"
```

Comment:
Is this path similar to the DPDK checkup?

Reply:
It is ([here](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/9ecc55e13b1aa5167ab25d5407650672739e2dc6/pkg/internal/config/config.go#L67))